requirements/test.txt

```diff
@@ -3,10 +3,19 @@ ddt
 freezegun
 selenium==2.52.0
 diff-cover
-django-nose==1.4.1
 django-statici18n==1.8.2
 edx-i18n-tools==0.4.7
 
--e git+https://github.com/edx/xblock-sdk.git@c40b5dc3e9a2ded54119e922edf486d3ccec6c35#egg=xblock_sdk
--e git://github.com/nosedjango/nosedjango.git@v1.0.12#egg=nosedjango-dev==v1.0.12
--e git+https://github.com/edx/edx-notifications.git@0.6.0#egg=edx-notifications==0.6.0
+-e git+https://github.com/edx/xblock-sdk.git@v0.1.6#egg=xblock_sdk
+-e git+git://github.com/nosedjango/nosedjango.git#egg=nosedjango
+-e git+https://github.com/edx/edx-notifications.git@1.0.4#egg=edx-notifications==1.0.4
+
+django-pyfs==1.0.6
+mako
+bok_choy==0.4.3
+python-dateutil==2.8.1
+pypng
+Django==1.11
+django-appconf==1.0.3
+django-nose==1.4.1
+pytest
```

Comment:
I think here are some things you can do:

1. This block is currently working on ironwood, so that means it should support the same version of requirements that ironwood does. You can check the packages installed in edx-solutions/edx-platform itself to see which versions to use here: https://github.com/edx-solutions/edx-platform/tree/master/requirements/edx
2. Likewise, for Juniper you can look at the version of packages installed here: https://github.com/edx-solutions/edx-platform/tree/rebase-juniper/requirements/edx
3. You can add version markers to install different versions of some packages on Python 2 vs Python 3 when running tests, here is an example from the chat block I upgraded recently: https://github.com/open-craft/xblock-chat/pull/40/files#diff-979cba9d30281710825398788016ac05
4. You can get rid of some requirements like django-nose etc but switching over to pytest. I also did that in the above PR.